projects/1.16/assets/pams-harvestcraft-2-food-extended/pamhc2foodextended/lang/zh_cn.json

```
@@ -33,23 +38,24 @@
   "item.pamhc2foodextended.cookedtofickenitem": "熟豆腐鸡肉",
   "item.pamhc2foodextended.cookedtofishitem": "熟豆腐鱼",
   "item.pamhc2foodextended.cookedtofuttonitem": "熟豆腐羊肉",
+
   "item.pamhc2foodextended.anchovypepperonipizzaitem": "鳀鱼香肠披萨",
   "item.pamhc2foodextended.bbqsauceitem": "烧烤酱",
-  "item.pamhc2foodextended.bbqchickenpizzaitem": "BBQ鸡肉披萨",
+  "item.pamhc2foodextended.bbqchickenpizzaitem": "烧烤鸡肉披萨",
   "item.pamhc2foodextended.beansandriceitem": "豆饭",
   "item.pamhc2foodextended.chimichangaitem": "墨西哥炸卷饼",
   "item.pamhc2foodextended.chipsandsalsaitem": "玉米片配萨尔萨酱",
   "item.pamhc2foodextended.cornmealitem": "玉米面",

```

```suggestion
  "item.pamhc2foodextended.cornmealitem": "玉米粉",
```
或是`粗玉米粉`（据维基百科）。玉米面应该是`Corn flour`，其质地较meal应该更为细腻。